lib/browser/rpc-server.js

```diff
@@ -383,7 +387,20 @@ ipcMain.on('ELECTRON_BROWSER_ASYNC_CALL_TO_GUEST_VIEW', function (event, context
     }
     guest[method].apply(guest, args)
   } catch (error) {
-    event.returnValue = exceptionToMeta(event.sender, contextId, error)
+    console.error(`${error}`)
```

Comment:
You need to send a reply back with the serialised exception here

Reply:
done